tests/rustdoc/strikethrough-in-summary.rs

```diff
@@ -1,6 +0,0 @@
-#![crate_name = "foo"]
```

Comment:
I've moved this check to the other test file, because they're otherwise identical (https://github.com/rust-lang/rust/pull/111152/files#diff-d7023b89dcafe34a2d98341780db733e68362bc533a2755fbef0c959eadbddf0R3).

Should I split this test again?

Reply:
Sorry I missed that. Instead, can you write a short comment at the top of the test file explaining what it's testing please?